packages/blockchain-wallet-v4-frontend/src/scenes/Help/index.tsx

```diff
@@ -51,15 +52,7 @@ const IconTextRow = styled.div`
   cursor: pointer;
 `
 
-const CircleBackground = styled.div`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  width: 48px;
-  height: 48px;
-  min-width: 48px;
-  background-color: ${(props) => props.theme.blue000};
-  border-radius: 48px;
+const StyledCircle = styled(CircleBackground)`
   margin: 8px 16px 8px 0;
```

Comment:
should we use pixels?

Reply:
Saw pixel in everywhere, in this case just create a styled component with the margin that already had